src/components/refresh/RefreshAlloactorButton.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from '@/components/ui/button'
+
+interface RefreshAlloactorButtonProps {
```

Comment:
```suggestion
interface RefreshAllocatorButtonProps {
```

Also in the file name